code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -184,3 +184,103 @@
 	..()
 	_color = null
 	update_icon()
+
+/obj/item/weapon/melee/energy/hfmachete
+	name = "high-frequency machete"
+	desc = "A high-frequency broad blade used either as an implement or in combat like a short sword."
+	icon_state = "hfmachete0"
+	var/base_state = "hfmachete"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	sharpness_flags = SHARP_BLADE | SERRATED_BLADE | CHOPWOOD
+	force = 13 // You can be crueler than that, Jack.
+	throwforce = 20
+	throw_speed = 8
+	throw_range = 8
+	w_class = W_CLASS_MEDIUM
+	flags = FPRINT
+	siemens_coefficient = 1
+	origin_tech = Tc_COMBAT + "=3" + Tc_SYNDICATE + "=3"
+	attack_verb = list("attacks", "dices", "cleaves", "tears", "cuts", "slashes",)
+
+/obj/item/weapon/melee/energy/hfmachete/update_icon()
+	if(active)
+		icon_state = "[base_state][active]"
```

Comment:
Not sure why you need an if here, since both ends of the if are exactly the same.